doc/source/ray-air/examples/tfx_tabular_train_to_serve.ipynb

```diff
@@ -0,0 +1,2059 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "VaFMt6AIhYbK"
+   },
+   "source": [
+    "This notebook is adapted from \"components keras\" [tutorial](https://www.tensorflow.org/tfx/tutorials/tfx/components_keras).\n",
```

Comment:
This notebook is adapted from a [Keras tutorial](link).

It uses X dataset and a Y model to predict whether a trip may generate a big tip.